strapi/strapi
# fix: multiple requests locale

### What does it do?

List View page was doing individual API requests for every entry.

i18n bulkActions and i18n localizations Cell was making use of `useDocument` hook, which was triggering those requests.
This PR proposes a fix to prevent those calls.

Before this PR, this was the network tab:
![image](https://github.com/user-attachments/assets/9de0f21e-7feb-4a37-89de-d11d941fa4b0)


After this PR:
![image](https://github.com/user-attachments/assets/48b164cc-09fa-4fa7-ba1b-16876160adfd)


### Why is it needed?

Describe the issue you are solving.

### How to test it?

- Have more than 1 locale configured 
- Access ListView of a Content type with i18n enabled
- See individual api requests made for each entry


### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
